src/TableHeader.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function TableHeader() {
+  return (
+    <thead>
+      <tr>
+        <th scope="col">#</th>
```

Comment:
Again here you could think about another way of rendering each of the columns without repeatedly writing out the `<th>...</th>` each time